packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -232,6 +233,38 @@ describe("Garbage Collection Tests", () => {
 		);
 	});
 
```

Comment:
Can you put these in a `describe` block now that there's more than 1 test doing stuff with sessionExpiry and container close?